mqtt-clock.c

```diff
@@ -181,6 +181,9 @@ struct mosqagent_result* clock_idle(struct mosqagent *agent)
 int main(int argc, char *argv[]) {
     int ret;
 
+    struct mqtta_memory_object mo;
+    mqtta_mo_ptr(&mo);
+
```

Comment:
`mqtta_mo_ptr()` returns the pointer stored in the memory object. You didn't set a pointer and you ignore the result. This call makes no sense.

Reply:
Yes, these are left-overs from some testing and will be evicted.